eventdata/challenges/bulk-size-evaluation.json

```diff
@@ -111,5 +111,59 @@
       "clients": {{ p_bulk_indexing_clients }}
     }
   ]
+},
+{
+  "name": "bulk-size-concurrency-evaluation",
+  "description": "Indexes with different bulk sizes and number of connections (2, 20, and 200) while keeping the number of documents in flight constant. IDs are autogenerated by Elasticsearch, meaning there are no conflicts.",
+  "meta": {
+    "benchmark_type": "indexing_bulksize"
+  },
+  "schedule": [
+    {
+      "operation": "delete-index"
+    },
+    {
+      "operation": {
+        "operation-type": "create-index",
+        "settings": {
+            "index.number_of_replicas": {{ p_replica_count }},
+            "index.number_of_shards": {{ p_shard_count }}
+        }
+      }
+    },
+    {
+      "name": "index-append-1000-2clients",
```

Comment:
I wonder whether it would make sense to use Jinja templating here but I'd also be ok as it is written now as well.